### ISSUE TITLE
ReDet only supports ReFPN neck

### ISSUE THREAD

**Issue by mmoghadam11** [csuhan/ReDet#93]
hi @csuhan
in detector config you put this:
`assert neck['type'] == 'ReFPN', 'ReDet only supports ReFPN neck'`

i think its for using this fpn for extracting the `re_resnet50`

but can i put attention like `BFP` after `ReFPN` ???

**Comment by csuhan** [comment]
Yes. You can comment this line. However, BFP (with regualr conv) may break the rotation equivariance and invariance. You can try e2cnn-based BFP, just like our ReFPN.
